frontend/src/app/draft/page.tsx

```diff
@@ -0,0 +1,102 @@
+"use client";
+
+import { axiosInstance } from "@services/api-client";
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { Heading } from "@ui/Heading";
+import { LinkButton } from "@ui/LinkButton";
+
+interface dataValues {
+  id: number;
+  title: string;
+  desc: string;
+  author: string;
+}
+
+const Draft = () => {
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    async function getBooks() {
+      const response = await axiosInstance.get("/api/books/10/pages");
+      setData(response.data);
+    }
+    getBooks();
+  }, []);
+
+  return (
+    <section className="w-full py-10 mt-5 bg-[url('/assets/bg_ellipse.png')] bg-no-repeat bg-cover bg-right">
```

Comment:
It would be easy to customize the tailwind.config file to get the background image.